Fleece/Support/ParseDate.cc

```diff
@@ -558,6 +558,8 @@ namespace fleece {
         auto offset = seconds(-s);
 #elif defined(__DARWIN_UNIX03) || defined(__ANDROID__) || defined(_XOPEN_SOURCE) || defined(_SVID_SOURCE)
         auto offset = seconds(-timezone);
+#elif defined(__EMSCRIPTEN__)
+        auto offset = seconds(timezone);
```

Comment:
Is this negated on purpose?  Does ECMAScript somehow change the functionality here?

Reply:
It was necessary to make the tests pass, but it seems like a bug in Emscripten which implements all these UNIX APIs. Not sure what the current behavior is, though.